janus_server/src/datastore.rs

```diff
@@ -1059,7 +1059,7 @@ impl Transaction<'_> {
                 WHERE
                     task_id = (SELECT id FROM tasks)
                     AND unit_interval_start >= $2
-                    AND (unit_interval_start + (SELECT min_batch_duration FROM tasks) * interval '1 second') < $3
+                    AND (unit_interval_start + (SELECT min_batch_duration FROM tasks) * interval '1 second') <= $3
```

Comment:
I want to draw attention to this change: the intent here is that a batch unit with interval `[100, 200)` should be considered included in the batch interval `[0, 200)`.